tests/docker/Dockerfile

```diff
@@ -35,26 +35,26 @@ RUN /tool-pkgs && \
 
 #################################
 
-FROM base as kafka-streams-examples
+FROM base AS kafka-streams-examples
 
 # Install kafka streams examples.  This is a very slow step (tens of minutes), doing
 # many maven dependency downloads without any parallelism.  To avoid re-running it
-# on unrelated changes in other steps, this step is as early on the Dockerfile as possible.
+# on unrelated changes in other steps, this step is AS early on the Dockerfile AS possible.
```

Comment:
too much search and replace here?

Reply:
Oops yup, scanned for this but apparently jumped right over it.

Reply:
Fixed in push.